config/__init__.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+FLASK_ENV = os.environ.get("FLASK_ENV")
+
+if not FLASK_ENV:
+    raise KeyError("FLASK_ENV does not exist in environ")
+
+match FLASK_ENV:
+    case "development":
+        from config.environments.development import (  # noqa
+            DevelopmentConfig as Config,  # noqa
+        )  # noqa
+    case "unit_test":
+        from config.environments.unit_test import (  # noqa
+            UnitTestConfig as Config,  # noqa
+        )  # noqa
```

Comment:
⛏️ We can put `#flask8 : noqa ` at the top of the file if we want to avoid the no qa's.